### PR TITLE
can i get my pizza to people component into the mainline code

### PR DESCRIPTION
**Pull request by melsummy**

### CODE REVIEW

**Comment by tsteelematc** [comment]
You have merge conflicts and have to merge the latest master into your PR branch and resolve... And you chose a different theme, so make sure to resolve appropriately and keep the theme of master...

Hope this makes sense, if not, let me know and I can help.

**Comment by melsummy** [comment]
@tsteelematc 
i do believe i have fixed the conflicts

**Comment by tsteelematc** [comment]
Could you move your component to the bottom...

And not make changes to the original Pizza Toppings component in master...

Hope that makes sense. Let me know if you have questions...

image

**Comment by melsummy** [comment]
@tsteelematc 
I think i fixed the problem
I did have some trouble as I did something, that I'm not really sure how but I think I fixed it. 
otherwise please let me know

File: src/app/pizza-toppings/pizza-toppings.component.html [hunk 1]
```diff
@@ -33,6 +33,9 @@
...
             Total: {{ totalPrice | currency }}
-        </h3>        
+        </h3>
     </mat-card-content>
-</mat-card>
\ No newline at end of file
+</mat-card>
+
+
+<app-pizza-calculator></app-pizza-calculator>
```
Comment (tsteelematc) [inline src/app/pizza-toppings/pizza-toppings.component.html]: I think you want to put this in app.component.html, not in the pizza-toppings components

Comment (melsummy) [reply src/app/pizza-toppings/pizza-toppings.component.html]: @tsteelematc 
done

**Comment by melsummy** [comment]
@tsteelematc 
ok i think i get it now

**Comment by melsummy** [comment]
@tsteelematc 
comments out the CSS

**Pull request closed without merging**